plenum/server/node.py

```diff
@@ -3487,6 +3529,9 @@ def onBatchRejected(self, ledger_id):
             self.get_req_handler(ledger_id).onBatchRejected()
         else:
             logger.debug('{} did not know how to handle for ledger {}'.format(self, ledger_id))
+
+        self.audit_handler.post_batch_rejected()
```

Comment:
ledger_id required, if we still  want to use it. (probable test fault reason)

Reply:
Yes, thank you